### ISSUE TITLE
Release notes version 1.1

### ISSUE THREAD

**Issue by stijnwop** [stijnwop/hoseSystem#48]

**Comment by TomCat67** [comment]
Don't know if this is the right spot - but has experienced something odd about release 1.1 .. The zip version on modhub (who hasn't got the HoseSystemVehicle.lua file attached in the zip) and the 1.1 download here (who has the HoseSystemVehicle.lua attached in the .modding folder) isn't the same .. Couldn't get the Hosesystem activated - it was being read, and nothing wrong in the game log, but the hosesystem wasn't working .. By using the 1.1 version here (download and unzip yourself) all is working quite nicely  .. Couldn't find anything about an issue about the modhub 1.1 version , so thought I might as well ask if somebody else has experienced the same .. TC67 ..

**Comment by stijnwop** [comment]
Yeah the modding folder and data stuff is not needed for the mod himself but it's in the github repository for source.

Actually the zip from modhub should just work fine.. specially when you can run the github version. Make sure you dont have it double. So really strange.. also make sure to activate etc because it probably will have a different hash.

**Comment by TomCat67** [comment]
What I missed, was the three lines in the log file, when buying a vehicle with the hosesystem : HoseSystem [Notice] : {
|  hasHoseSystemFillArm : true
|  hasHoseSystemConnectors : true
|  hasHoseSystemPumpMotor : true
}
And just to be sure it was the vehicle, and not the map, I did the test on the Lossberg map - thought it was a bit strange, that when using the modhub 1.1 version, I wasn't getting these lines about the hosesystem being recognized - and the hosesystem apparently didn't work .. But when using the GitHub 1.1 version, all the lines was in the log, and I could get the vehicle to fill from the pigmanurepit with the fillarm (with some trouble, but that was mainly because of some collision issues between the fillarm an the pit ..) .. TC67

**Comment by stijnwop** [comment]
All the stuff that's being printed in your log and visual colored lines etc.. is for debug propose and only for the github version. This is not present on the released version in modhub since I disabled that because I don't want to spam people's log as you can imagine.

**Comment by TomCat67** [comment]
Ahh ... Okay - makes sense .. But nice to have when modding maps and vehicles - never know in which end it's wrong, when it doesn't work as intended .. I'll stick with the GitHub version in the future (as long as it's updated here ..) Thanks - TC67
